src/Plugin/migrate/process/TransformParentValue.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Drupal\multiversion\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+
+/**
+ * Perform custom value transformations.
+ *
+ * @MigrateProcessPlugin(
+ *   id = "transform_parent_value"
+ * )
+ *
+ * To do custom value transformations use the following:
+ *
+ * @code
+ * field_text:
+ *   plugin: transform_parent_value
+ *   source: text
+ * @endcode
+ */
+class TransformParentValue extends ProcessPluginBase {
```

Comment:
TransformMenuLinkContentParentValue - so it would be cleared what plugin is doing.